NetSpell.SpellChecker/Spelling.cs

```diff
@@ -619,7 +619,7 @@ public int EditDistance(string source, string target, bool positionPriority)
         /// </remarks>
         public int EditDistance(string source, string target)
         {
-            return EditDistance(source, target, true);
+            return EditDistance(source, target, positionPriority: true);
```

Comment:
Was this a preparation for some other change?

Reply:
No, it is purely cosmetic and snuck in by mistake. Can remove if you like, though it's harmless enough.

Reply:
OK

Reply:
Let's keep unrelated changes separate